deepchecks/tabular/dataset.py

```diff
@@ -535,9 +535,17 @@ def train_test_split(self: TDataset,
         return self.copy(train_df), self.copy(test_df)
 
     @staticmethod
-    def _infer_label_type(label_col: pd.Series):
+    def _infer_label_type(label_col: pd.Series, infer_task_type_from_label_nunique: int = 10):
```

Comment:
As this attribute is used for raising a warning only, there's not need for it to be a parameter of the function.